frontend/api/v1_1/routes.py

```diff
@@ -46,7 +44,17 @@ def define_routes(application):
     # tags routes
     application.route("/tags",
                       callback=tags.list_available_tags)
+    # files routes
+    application.route("/files",
+                      callback=files.list)
+    # files routes
+    application.route("/files/<sha256>/download",
+                      callback=files.download)
+    application.route("/files/<sha256>/results",
+                      callback=files.results)
+    application.route("/files/<sha256>/scans",
+                      callback=files.scans)
     application.route("/files/<sha256>/tags/<tagid>/add",
```

Comment:
Hum, I didn't see that before. To fit already existing routes, we should do a `POST Request on /tags`, with inside the request the tag name. In fact, we don't have `add` route, only use POST request to simulate the add. I don't think it's the scope of this PR, but maybe next time (or if you've got enough time, but i've got some doubts about that).